pennylane_lightning/core/src/simulators/lightning_gpu/bindings/LGPUBindings.hpp

```diff
@@ -150,6 +150,8 @@ void registerBackendClassSpecificBindings(PyClass &pyclass) {
             },
             py::arg("async") = false,
             "Initialize the statevector data to the |0...0> state")
+        .def("collapse", &StateVectorT::collapse,
```

Comment:
Does LGPU support `normalize`? It shouldn't be difficult to add to the API at some point

Reply:
Thanks @maliasadi ! Is it a feature/method required by the pennylane frontend? The `custatevecCollapseOnZBasis` API used in the `collapse` method does the zero and normalize work internally.